products/extensions/b2b-components/employee-management/concept/index.md

```diff
@@ -0,0 +1,15 @@
+---
+nav:
+  title: Concepts
```

Comment:
For future reference - defining the title is not needed if it matches the `h1`.

Reply:
@bojanrajh I mainly did it for position. I never came across a position without title. I guess this is custom from us, right? Did not find a reference to that syntax in vitepress.


This would work ?

```
---
nav:
  position: 40

---
```